src/Marten/Events/Daemon/AsyncDaemonHealthCheckExtensions.cs

```diff
@@ -0,0 +1,95 @@
+#nullable enable
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using JasperFx.Core;
+using Marten.Events.Projections;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+
+namespace Marten.Events.Daemon;
+
+public static class AsyncDaemonHealthCheckExtensions
+{
+    /// <summary>
```

Comment:
**COMMENT:** KUDOS for not forgetting about XML comments and adding code samples to them :+1:

Reply:
Thanks mate. Good XML comments is a must! :)